### PR TITLE
add AndroidSubscriptions (ported from RxAndroid)

### PR DESCRIPTION
**Pull request by gfx**
cc: @sys1yagi

### CODE REVIEW

File: rxt4a/src/main/java/com/cookpad/android/rxt4a/subscriptions/AndroidSubscriptions.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+     * @param unsubscribe An action called in unsubscribe.
+     */
+    public static Subscription unsubscribeOnMainThread(final Action0 unsubscribe) {
+        return Subscriptions.create(new Action0() {
+            @Override
+            public void call() {
+                if (Looper.getMainLooper() == Looper.myLooper()) {
+                    unsubscribe.call();
+                } else {
+                    Handler handler = new Handler(Looper.getMainLooper());
```
Comment (sys1yagi) [inline rxt4a/src/main/java/com/cookpad/android/rxt4a/subscriptions/AndroidSubscriptions.java]: I think that you should define Handler in static.

Comment (gfx) [inline rxt4a/src/main/java/com/cookpad/android/rxt4a/subscriptions/AndroidSubscriptions.java]: Good catch. Fixed in b2bccf2

File: README.md [hunk 2]
```diff
@@ -54,11 +54,24 @@ This is a variation of `CompositeSubscription` but can be reused multiple times.
...
 
+## AndroidSubscriptions
+
+```java
+Subscription subscription = observable.subscribe(subscriber);
+
+subscription.add(AndroidSubscriptions.unsubscribeOnMainThread(new Action0() {
+    @Override
+    public void call() {
+        // touch views
```
Comment (sys1yagi) [inline README.md]: This method is called when parent subscription was unsubscribed. not onNext() receive.

**Comment by sys1yagi** [comment]
[LGTM](http://lgtm.in/i/ahMejkXkq)

**Pull request merged**